>From 4c6bbd75f070ebd55a889868182891fa9adacc0e Mon Sep 17 00:00:00 2001
From: Michael Biebl <biebl@debian.org>
Date: Sun, 21 Jul 2013 17:26:58 +0200
Subject: [PATCH] Test for usb-db before running it

In newer udev versions, usb-db has been replaced by the hwdb builtin.
Instead of removing the import, test for the helper before running it.
This way we are compatible with both old and new udev.
---
 rules/69-cd-sensors.rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rules/69-cd-sensors.rules b/rules/69-cd-sensors.rules
index a955a63..5bc8cdb 100644
--- a/rules/69-cd-sensors.rules
+++ b/rules/69-cd-sensors.rules
@@ -87,7 +87,7 @@ ATTRS{idVendor}=="273f", ATTRS{idProduct}=="1002", ENV{COLORD_SENSOR_KIND}="colo
 # color calibration device
 ENV{COLORD_SENSOR_KIND}=="*?", ENV{COLOR_MEASUREMENT_DEVICE}="1"
 ENV{COLORD_SENSOR_KIND}=="*?", ENV{ID_MODEL}=="", IMPORT{program}="usb_id --export %p"
-ENV{COLORD_SENSOR_KIND}=="*?", ENV{ID_MODEL_FROM_DATABASE}=="", IMPORT{program}="usb-db %p"
+ENV{COLORD_SENSOR_KIND}=="*?", ENV{ID_MODEL_FROM_DATABASE}=="", TEST=="/lib/udev/usb-db", IMPORT{program}="usb-db %p"
 
 # Allow the daemon to access the color devices
 ENV{COLORD_SENSOR_KIND}=="*?", GROUP="root"
-- 
1.8.3.2

